src/main/java/edu/cornell/kfs/concur/batch/ConcurStandardAccountingExtractToPdpStep.java

```diff
@@ -44,53 +47,96 @@ public boolean execute(String jobName, Date jobRunDate) throws InterruptedExcept
 
     protected boolean processCurrentFileAndExtractPdpFeedFromSAEFile(File currentFile) {
         boolean success = true;
-        if (!currentFile.isDirectory()) {
+        if (!currentFile.isDirectory() && isTextFile(currentFile)) {
             LOG.debug("processCurrentFileAndExtractPdpFeedFromSAEFile, current File: " + currentFile.getName());
             try {
                 ConcurStandardAccountingExtractFile concurStandardAccoutingExtractFile = getConcurStandardAccountingExtractService()
```

Comment:
I just regularly use the getters when using services instead of directly referencing the services themselves.  Do you think there is a problem doing that?